src/app/main/component/events/components/event-date-time-picker/event-date-time-picker.component.ts

```diff
@@ -284,6 +310,8 @@ export class EventDateTimePickerComponent implements OnInit, OnChanges {
     const checkTime = time.split(':');
```

Comment:
if (!curTime && checkTime[0] === '00') return
this.timeArrStart = curTime ? [...this.timeArr.slice(curTime + 1, +checkTime[0])] : [...this.timeArr.slice(0, +checkTime[0])]